types/koa-logger/index.d.ts

```diff
@@ -3,18 +3,19 @@
 // Definitions by: Joshua DeVinney <https://github.com/geoffreak>
 //                 Tomek Łaziuk <https://github.com/tlaziuk>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// Minimum TypeScript Version: 4.0
```

Comment:
`types/koa` has `// TypeScript Version: 3.0` itself.
what's is here to gain really by using named tuples?
If this provides some useful gains, you'd probably have to 
split implementation for <=4.0:
https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-very-new-typescript-versions

Reply:
It is nicer to have named tuples as it provides a better typing experience, however as mentioned below, maybe it's worth just fixing the type for now